Mathlib/MeasureTheory/Integral/CompactlySupported.lean

```diff
@@ -0,0 +1,51 @@
+/-
+Copyright (c) 2025 Yongxi Lin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yongxi Lin, Thomas Zhu
+-/
+import Mathlib.MeasureTheory.Function.LocallyIntegrable
+import Mathlib.MeasureTheory.Integral.Bochner.Basic
+import Mathlib.Topology.ContinuousMap.CompactlySupported
+
+/-!
+# Integrating compactly supported continuous functions
+
+This file contains definitions and lemmas related to integrals of compactly supported continuous
+functions.
+-/
+
+open scoped ENNReal NNReal
+open CompactlySupported MeasureTheory
+
+variable {X : Type*}
+
+namespace CompactlySupportedContinuousMap
+
+lemma integrable {X E : Type*} [MeasurableSpace X]
+    [TopologicalSpace X] [NormedAddCommGroup E] (f : C_c(X, E))
+    {μ : Measure X} [OpensMeasurableSpace X] [IsFiniteMeasureOnCompacts μ] :
+    Integrable f μ :=
+  f.continuous.integrable_of_hasCompactSupport f.hasCompactSupport
+
+variable [TopologicalSpace X] [T2Space X] [LocallyCompactSpace X] [MeasurableSpace X] [BorelSpace X]
+variable (Λ : C_c(X, ℝ) →ₚ[ℝ] ℝ)
+
+/-- Integral as a positive linear functional on `C_c(X, ℝ)`. -/
+@[simps!]
+noncomputable def integralPositiveLinearMap (μ : Measure X) [OpensMeasurableSpace X]
```

Comment:
Why are you assuming `OpensMeasurableSpace` when you already have `BorelSpace` above? Can you clarify which assumption is needed?